scripts/toolboxSrc/applyEnvVarsToClientAssets.ts

```diff
@@ -60,6 +60,7 @@ const rewriteIndexHTML = () => {
     stripe: process.env.STRIPE_PUBLISHABLE_KEY,
     publicPath: __webpack_public_path__,
     oauth2Redirect: process.env.OAUTH2_REDIRECT,
+    openAI: process.env.OPEN_AI_API_KEY,
```

Comment:
-2 please correct me if i'm wrong, but isn't this a secret key?

Reply:
It is - we could make `openAI` a boolean and check if the key exists?